dashboard/public/frequency/frequency.js

```diff
@@ -0,0 +1,879 @@
+
+var ALPHABET = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+var LETTERS = ALPHABET.split('');
+var ENGLISH = {
+  A: 0.08167,
+  B: 0.01492,
+  C: 0.02782,
+  D: 0.04253,
+  E: 0.12702,
+  F: 0.02228,
+  G: 0.02015,
+  H: 0.06094,
+  I: 0.06966,
+  J: 0.00153,
+  K: 0.00772,
+  L: 0.04025,
+  M: 0.02406,
+  N: 0.06749,
+  O: 0.07507,
+  P: 0.01929,
+  Q: 0.00095,
+  R: 0.05987,
+  S: 0.06327,
+  T: 0.09056,
+  U: 0.02758,
+  V: 0.00978,
+  W: 0.02361,
+  X: 0.0015,
+  Y: 0.01974,
+  Z: 0.00074
+};
+
+// Debounce function, courtesy of
+// http://davidwalsh.name/function-debounce
+// Currently unused.
+var debounce = function (func, wait, immediate) {
```

Comment:
I wonder if we should just include lodash in Dashboard as well?  Nit: he doesn't publish a clear license for this code snippet.